src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/staves/StormStaff.java

```diff
@@ -36,12 +41,16 @@
     public static final int MAX_USES = 8;
 
     private static final NamespacedKey usageKey = new NamespacedKey(Slimefun.instance(), "stormstaff_usage");
+    private final ItemSetting<Boolean> allowNonPVPUse = new ItemSetting<>(this, "allow-non-pvp-use", false);
+
+    public static final Map<UUID, UUID> stormStaffLightnings = new HashMap<>();
```

Comment:
No. Do not use public global variables NotLikeThis

Make a getter and also a make a comment above explaining what the key and value is since it isn't clear